environments/aat.yml

```diff
@@ -134,9 +134,9 @@ cname:
   - name: "immigration-appeal"
     ttl: 3600
     record: "hmcts-aat.azurefd.net"
-  - name: "afdverify.idam-web-public"
+  - name: "afdverify.idam-web-public-aat2"
```

Comment:
not sure why you're putting aat in these domains, but i've seen it going all over the place so won't make you change it here 😉

Reply:
It's causing confusion for some service teams and we don't want to switch yet. 

```
# Internal DNS
~/src/moj/idam-tools SIDM-3811
❯ dig idam-web-public.aat.platform.hmcts.net

; <<>> DiG 9.10.6 <<>> idam-web-public.aat.platform.hmcts.net
;; global options: +cmd
;; Got answer:
;; ->>HEADER<<- opcode: QUERY, status: NOERROR, id: 10239
;; flags: qr aa rd ra; QUERY: 1, ANSWER: 4, AUTHORITY: 4, ADDITIONAL: 5

;; OPT PSEUDOSECTION:
; EDNS: version: 0, flags:; udp: 4096
;; QUESTION SECTION:
;idam-web-public.aat.platform.hmcts.net.	IN A

;; ANSWER SECTION:
idam-web-public.aat.platform.hmcts.net.	300 IN CNAME hmcts-idam-web-public-idam-aat.trafficmanager.net.
hmcts-idam-web-public-idam-aat.trafficmanager.net. 100 IN CNAME	tmidam-web-public.aat.platform.hmcts.net.
tmidam-web-public.aat.platform.hmcts.net. 300 IN CNAME d060dd65-4af6-42e7-9e1d-adfd3b847879.cloudapp.net.
d060dd65-4af6-42e7-9e1d-adfd3b847879.cloudapp.net. 4 IN	A 51.145.33.254

;; AUTHORITY SECTION:
cloudapp.net.		96226	IN	NS	ns1prod.18.azuredns-prd.info.
cloudapp.net.		96226	IN	NS	ns1prod.18.azuredns-prd.org.
cloudapp.net.		96226	IN	NS	ns2prod.18.azuredns-prd.org.
cloudapp.net.		96226	IN	NS	ns2prod.18.azuredns-prd.info.

;; ADDITIONAL SECTION:
ns2prod.18.azuredns-prd.info. 2370 IN	A	13.107.160.201
ns1prod.18.azuredns-prd.info. 2984 IN	A	13.107.24.201
ns2prod.18.azuredns-prd.org. 993 IN	A	64.4.48.201
ns1prod.18.azuredns-prd.org. 601 IN	A	40.90.4.201

;; Query time: 42 msec
;; SERVER: 172.16.0.10#53(172.16.0.10)
;; WHEN: Mon Mar 09 15:05:23 GMT 2020
;; MSG SIZE  rcvd: 426


# Public DNS
~/src/moj/idam-tools SIDM-3811
❯ dig idam-web-public.aat.platform.hmcts.net

; <<>> DiG 9.10.6 <<>> idam-web-public.aat.platform.hmcts.net
;; global options: +cmd
;; Got answer:
;; ->>HEADER<<- opcode: QUERY, status: NOERROR, id: 210
;; flags: qr rd ra; QUERY: 1, ANSWER: 5, AUTHORITY: 0, ADDITIONAL: 1

;; OPT PSEUDOSECTION:
; EDNS: version: 0, flags:; udp: 512
;; QUESTION SECTION:
;idam-web-public.aat.platform.hmcts.net.	IN A

;; ANSWER SECTION:
idam-web-public.aat.platform.hmcts.net.	3599 IN	CNAME hmcts-aat.azurefd.net.
hmcts-aat.azurefd.net.	1799	IN	CNAME	t-0001.t-msedge.net.
t-0001.t-msedge.net.	59	IN	CNAME	Edge-Prod-LON21r3.ctrl.t-0001.t-msedge.net.
Edge-Prod-LON21r3.ctrl.t-0001.t-msedge.net. 239	IN CNAME standard.t-0001.t-msedge.net.
standard.t-0001.t-msedge.net. 239 IN	A	13.107.246.10

;; Query time: 80 msec
;; SERVER: 8.8.8.8#53(8.8.8.8)
;; WHEN: Mon Mar 09 15:05:32 GMT 2020
;; MSG SIZE  rcvd: 205
```

Reply:
idam-web-public2 would have the same outcome 😉